tasks/gulp/__tests__/after-build-package.test.js

```diff
@@ -61,10 +61,26 @@ describe('package/', () => {
           filesNotInSrc = glob.sync('{' + additionalFilesNotInSrc.join(',') + '}', { cwd: 'package' })
 
           return files
-            // Remove /src prefix from filenames
-            .map(file => file.replace(/^src\//, ''))
+            .map(file => {
```

Comment:
Nice!

I guess in the future it'd be nice to test the bundling somehow as well. Maybe implement a basic bundle test and make sure it's not changing size too greatly, or repeating code, etc. Could argue that's more on the user for their specific implementation, but I think it'd give us a bit of confidence if we had these kind of insights into bundled code.

Reply:
Agreed, this testing is very comprehensive - it gives us a kind of very basic level of reassurance that things are working vaguely as expected. I wasn't sure how any new kind of testing like that would 1) work (I've not come across that kind of testing before), or 2) fit into our current testing strategy, but I agree it would be really good to have.